src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --accent-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the font 'Roboto' is properly loaded in your HTML file, as it is being used as the primary font-family here.